src/vs/workbench/contrib/workspace/browser/workspace.contribution.ts

```diff
@@ -111,52 +111,50 @@ export class WorkspaceTrustRequestHandler extends Disposable implements IWorkben
 			];
 
 			// Dialog
-			const result = await this.dialogService.show(
-				Severity.Info,
-				this.workspaceContextService.getWorkbenchState() !== WorkbenchState.EMPTY ?
+			await this.dialogService.prompt<void>({
```

Comment:
@lszomoru fyi please check, this one is a bit more complex adoption of the new dialog api